### PR TITLE
Styling of layout

### PR DESCRIPTION
**Pull request by lollypop27**
Added styles to main layout page
Added styles to login and create account page
Add styles to dish display page

### CODE REVIEW

**Comment by tochman** [comment]
Can we see some screen shots?

**Comment by diraulo** [comment]
@tochman there you go
screenshot 2016-07-01 13 17 17

**Comment by tochman** [comment]
Have a look at this gist to add sticky footer: https://gist.github.com/tochman/4578ca634c9de5051ee2

**Comment by diraulo** [comment]
Thanks @tochman

**Comment by tochman** [comment]
Are you guys happy with this? Ready to merge?

**Comment by lollypop27** [comment]
No not ready to merge @tochman, just went through the code with @MikaelFeher  and need to do some more syle attributes

**Comment by lollypop27** [comment]
@tochman ready to merge CSS

File: lib/views/contact.erb [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <div class="row">
   <div class="medium-6 medium-centered large-4 large-centered columns">
     <h2>Contact Us<h2>
-    <ul>
-      <li>Tel: 011 555 555</li>
-      <li>Email: info@ciaobaby.com</li>
-      <li>Address: The Mall, 1 Drive Avenue, Pretoria</li>
-    </ul>
+      <ul style="font-size: 16px; font-weight: normal; color:#000;">
```
Comment (tochman) [inline lib/views/contact.erb]: I would extract tis to a css class.

**Comment by lollypop27** [comment]
@tochman cool have fixed the UL style in CSS sheet and taken out the hardcoded style in the UL tag.
you can merge. Thanks

**Pull request merged**